index.html

```diff
@@ -1167,17 +1165,16 @@ <h4>
       amount of milliseconds.
     </p>
     <pre class="example">
-      const reader = new NDEFReader();
-
       function read() {
         return new Promise((resolve, reject) => {
           const ctlr = new AbortController();
           ctlr.signal.onabort = reject;
-          reader.addEventListener("reading", event => {
+          navigator.nfc.ndef.addEventListener("reading", event => {
             ctlr.abort();
```

Comment:
I'm not sure about this. How does this work exactly?

Reply:
@kenchris Do we need `catch(err => reject(err));`?

Reply:
Let's keep it like this.

Reply:
I think we need to remove the ctlr.signal.onabort because you don't need to reject if it actually succeeds getting a reading.

Reply:
It works the way that if you get an event it resolves with the data and aborts the reader (stops reading).

if it fails for some reason, it will just reject

Reply:
it might actually be working today with ctlr.signal.onabort = reject; because abort is running async code and only one of reject/resolve can be called (the next is ignored)

Reply:
Maybe we should explain how it works :)

Reply:
Please open a new PR for this ;)